### ISSUE TITLE
Stuck on Installing GRUB...

### ISSUE THREAD

**Issue by Necklaces** [Mati365/pyWinUSB#4]
I tried seeing if any other issues like this came up, couldn't find any, sorry if this is a duplicate.

I installed with pip, ran with sudo, selected my windows iso, went with ntfs as that was hooked by default, selected my usb-stick and went apply. It ran fine until it said "Installing GRUB...", it's been like that for an hour now.

Any ideas?

**Comment by Mati365** [comment]
Please check console log, it seems to be a bug in grub-install linux package. Anyway program should have its own mbr creator. Did you try booting from usb drive after canceling?

**Comment by Necklaces** [comment]
Yeah I did try booting with that USB after canceling, but nothing happened, bios couldnt find anything and booted into my other drive.

However, rebooting my machine and trying pyWinUSB again worked! (although I had to wait for quite a while), there is a possibility I was too quick to jump to the conclusion that it was "hanging", or maybe the fact that I had been trying multiple things including gparted that session had something to do with it.

Anyway, sorry, might've been a corner case, I've just been trying for hours with different things, and now this finally worked! Thank you so much for making this!
